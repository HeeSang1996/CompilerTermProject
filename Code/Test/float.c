void main() {
    float a = 0.0;
    float b = 0.0123456789;
    float c = 0.0000000001;
    float d = 1.0;
    float e = 1234567890.0;
    float f = 1.0123456789;
    float g = 1.0000000001;
}