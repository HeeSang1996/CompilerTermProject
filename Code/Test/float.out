ID void
ID main
PAREN (
PAREN )
BRACE {
VARIABLE float
ID a
ASSIGN =
FLOAT 0.0
TERM ;
VARIABLE float
ID b
ASSIGN =
FLOAT 0.0123456789
TERM ;
VARIABLE float
ID c
ASSIGN =
FLOAT 0.0000000001
TERM ;
VARIABLE float
ID d
ASSIGN =
FLOAT 1.0
TERM ;
VARIABLE float
ID e
ASSIGN =
FLOAT 1234567890.0
TERM ;
VARIABLE float
ID f
ASSIGN =
FLOAT 1.0123456789
TERM ;
VARIABLE float
ID g
ASSIGN =
FLOAT 1.0000000001
TERM ;
VARIABLE float
ID h
ASSIGN =
FLOAT 0.0
TERM ;
VARIABLE float
ID i
ASSIGN =
FLOAT 0.0123456789
TERM ;
VARIABLE float
ID j
ASSIGN =
FLOAT 0.0000000001
TERM ;
VARIABLE float
ID k
ASSIGN =
FLOAT 1.0
TERM ;
VARIABLE float
ID l
ASSIGN =
FLOAT 1234567890.0
TERM ;
VARIABLE float
ID m
ASSIGN =
FLOAT 1.0123456789
TERM ;
VARIABLE float
ID n
ASSIGN =
FLOAT 1.0000000001
TERM ;
VARIABLE float
ID o
ASSIGN =
FLOAT 0.0
TERM ;
VARIABLE float
ID p
ASSIGN =
FLOAT 0.0123456789
TERM ;
VARIABLE float
ID q
ASSIGN =
FLOAT 0.0000000001
TERM ;
VARIABLE float
ID r
ASSIGN =
FLOAT 1.0
TERM ;
VARIABLE float
ID s
ASSIGN =
FLOAT 1234567890.0
TERM ;
VARIABLE float
ID t
ASSIGN =
FLOAT 1.0123456789
TERM ;
VARIABLE float
ID u
ASSIGN =
FLOAT 1.0000000001
TERM ;
BRACE }
